### PR TITLE
inclusion of data extractor for current/voltage/temperature vs time d…

### PR DESCRIPTION
**Pull request by npaulson**
inclusion of data extractor for current/voltage/temperature vs time data, and example processing script for open lead-acid battery cycling dataset. Corrections to postprocessing codes for extracting capacity and energy versus cycling

### CODE REVIEW

File: batdata/extractors/tIVT.py [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+    def group(self, files: Union[str, List[str]], directories: List[str] = None,
+              context: dict = None) -> Iterator[Tuple[str, ...]]:
+        for file in files:
+            if file.lower().endswith('.npy'):
+                yield file
+
+    def generate_dataframe(self, file: str, file_number: int = 0, start_cycle: int = 0,
+                           start_time: float = 0) -> pd.DataFrame:
+        """Generate a DataFrame containing the data in this file
```
Comment (WardLT) [inline batdata/extractors/tIVT.py:32]: You don't need to copy the docstring from a superclass, and should delete this for brevity.

File: batdata/extractors/tIVT.py [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+        add_method(df_out)
+        add_substeps(df_out)
+
+        # Add capacity and energy calculations
+        df_out = compute_capacity_energy(df_out)
+
+        return df_out
+
+    def implementors(self) -> List[str]:
+        return ['Noah, Paulson <npaulson@anl.gov>', 'Kubal, Joesph <kubal@anl.gov>', 'Ward, Logan <lward@anl.gov>']
```
Comment (WardLT) [inline batdata/extractors/tIVT.py:115]: If this was just you, you can delete Joe and I

File: scripts/howey.py [hunk 3]
```diff
@@ -0,0 +1,96 @@
+"""Parse tIVT .npy data from the Howey dataset
```
Comment (WardLT) [inline scripts/howey.py:1]: Could you group these two scripts in a single folder, then add a README to explain what they are for?

**Review by WardLT — CHANGES_REQUESTED** [review]
Just a few minor changes. Thanks, @npaulson !

File: batdata/extractors/tIVT.py (hunk 1 above)
Comment (npaulson) [reply batdata/extractors/tIVT.py:32]: Will do! Sorry for the delay, I didn't realize something was waiting on me!

**Review by WardLT — APPROVED** [review]

**Pull request merged**